node/actors/sync_blocks/src/lib.rs

```diff
@@ -48,6 +48,10 @@ impl SyncBlocks {
         storage: Arc<dyn WriteBlockStore>,
         config: Config,
     ) -> anyhow::Result<Self> {
+        if config.consensus_threshold == 0 {
+            tracing::warn!("QC validation is disabled; this is not secure");
+        }
```

Comment:
this introduces non-uniform interpretation of consensus_threshold, as for >0 all signatures are validated, even if there is more signatures than a threshold. IMO it would be better to introduce a separate parameter OR wrap consensus_thereshold into an Option<_>, so that the fact of ignoring the signatures is distinguished.